lib/super_spreader/stop_signal.rb

```diff
@@ -3,15 +3,21 @@
 require "redis"
 
 module SuperSpreader
+  # @deprecated Please use TrackBallast::StopSignal instead
   module StopSignal
+    # @deprecated Please use {TrackBallast::StopSignal.stop!} instead
     def stop!
+      warn "DEPRECATION WARNING: the class SuperSpreader::StopSignal is deprecated and will be removed in v1.0. " \
+        "Use TrackBallast::StopSignal instead."
       redis.set(stop_key, true)
     end
 
+    # @deprecated Please use {TrackBallast::StopSignal.go!} instead
     def go!
       redis.del(stop_key)
     end
 
+    # @deprecated Please use {TrackBallast::StopSignal.stopped?} instead
```

Comment:
When previewing the documentation using `rake doc`, these warnings appear:

```
[warn]: In file `lib/super_spreader/stop_signal.rb':7: Cannot resolve link to TrackBallast::StopSignal.go! from text:
	...{TrackBallast::StopSignal.go!}...
[warn]: In file `lib/super_spreader/stop_signal.rb':7: Cannot resolve link to TrackBallast::StopSignal.stop! from text:
	...{TrackBallast::StopSignal.stop!}...
[warn]: In file `lib/super_spreader/stop_signal.rb':7: Cannot resolve link to TrackBallast::StopSignal.stopped? from text:
	...{TrackBallast::StopSignal.stopped?}...
[warn]: In file `lib/super_spreader/stop_signal.rb':16: Cannot resolve link to TrackBallast::StopSignal.go! from text:
	...{TrackBallast::StopSignal.go!}...
[warn]: In file `lib/super_spreader/stop_signal.rb':9: Cannot resolve link to TrackBallast::StopSignal.stop! from text:
	...{TrackBallast::StopSignal.stop!}...
[warn]: In file `lib/super_spreader/stop_signal.rb':21: Cannot resolve link to TrackBallast::StopSignal.stopped? from text:
	...{TrackBallast::StopSignal.stopped?}...
```

It makes sense that it can't resolve the links because we are moving `StopSignal` to a different gem in https://github.com/doximity/track_ballast/issues/19.  When the curly braces are removed, the warnings go away and the resulting documentation appears to be identical.  What do you think about this change?

```suggestion
    # @deprecated Please use TrackBallast::StopSignal.stop! instead
    def stop!
      warn "DEPRECATION WARNING: the class SuperSpreader::StopSignal is deprecated and will be removed in v1.0. " \
        "Use TrackBallast::StopSignal instead."
      redis.set(stop_key, true)
    end

    # @deprecated Please use TrackBallast::StopSignal.go! instead
    def go!
      redis.del(stop_key)
    end

    # @deprecated Please use TrackBallast::StopSignal.stopped? instead
```

![Screenshot 2024-02-13 at 13 15 23](https://github.com/doximity/super_spreader/assets/5323/c0c03d31-5e48-40e4-b981-c086a4704263)